docs_src/path_params_numeric_validations/tutorial002.py

```diff
@@ -1,13 +1,14 @@
-from fastapi import FastAPI, Path
+from fastapi import FastAPI, Path, Query
 
 app = FastAPI()
 
 
 @app.get("/items/{item_id}")
 async def read_items(
-    q: str, item_id: int = Path(..., title="The ID of the item to get")
+    item_id: int = Path(..., title="The ID of the item to get"),
+    q: str = Query(None, alias="item-query"),
```

Comment:
I've added a suggestion, but I have the feeling this is related to the previous comment, am I right?
```suggestion
    q: Optional[str] = Query(None, alias="item-query"),
```